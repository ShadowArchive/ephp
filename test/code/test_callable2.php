<?php

//$result = $a[PCLZIP_CB_PRE_ADD](PCLZIP_CB_PRE_ADD, $data);

$a = [ "print_r", "var_dump" ];

const PRINT_R = 0;
const VAR_DUMP = 1;

$data = [1,2,3];

$result = $a[PRINT_R]($data);
$result = $a[VAR_DUMP](PCLZIP_CB_PRE_ADD, $data);

