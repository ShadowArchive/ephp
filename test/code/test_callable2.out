Array
(
    [0] => 1
    [1] => 2
    [2] => 3
)

Notice: Use of undefined constant PCLZIP_CB_PRE_ADD - assumed 'PCLZIP_CB_PRE_ADD' in {{CWD}}/test/code/test_callable2.php on line 13
string(17) "PCLZIP_CB_PRE_ADD"
array(3) {
  [0]=>
  int(1)
  [1]=>
  int(2)
  [2]=>
  int(3)
}
